lms/urls.py

```diff
@@ -237,6 +237,19 @@
     )
 
 if settings.COURSEWARE_ENABLED:
+    COURSE_URLS = patterns(
```

Comment:
@muhhshoaib I'll look into this.

Just trying to catch up, @clintonb is this your last remaining comment?